.gitignore

```diff
@@ -13,9 +13,9 @@ package-lock.json
 coverage
 
 # build
-locale
-plugin
+locale/
+plugin/
```

Comment:
what's this change for?

Reply:
@iamkun 
To include `src/locale` & `src/plugin` folder. 

I found that changes under `src/locale` are ignored when the `.gitignore` has `locale`. But once it changes to `locale`, it just ignores `locale` and stopped to ignore `src/locale`.

Reply:
Sure, got it thanks.